pom.xml

```diff
@@ -54,16 +54,22 @@
             <artifactId>vaadin-core</artifactId>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
-            <version>3.1.0</version>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>5.0.0</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-testbench</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-testbench-core-junit5</artifactId>
```

Comment:
Should it be `vaadin-testbench-junit5` now?

Reply:
Leaving it as it now is fine, but let's remember to update the platform with latest testbench, wait for new platform release and update it here in this demo.

Reply:
I think it is still not included in vaadin-bom

Reply:
`vaadin-testbench-junit5` is not released in stable version, only available in SNAPSHOT.
Here https://tools.vaadin.com/nexus/content/repositories/vaadin-prereleases/com/vaadin/vaadin-testbench-junit5/24.0-SNAPSHOT/ .

But after platform next release, we can use it.
`vaadin-testbench-junit5` dependes on `vaadin-bom`
`vaadin-bom` contains `vaadin-testbench-bom`
`vaadin-testbench-bom` includes `vaadin-testbench-core-junit5`.

So everything should work.